app/models/manageiq/providers/openstack/inventory/collector/target_collection.rb

```diff
@@ -254,7 +254,7 @@ def infer_related_cloud_tenant_ems_refs_db!
 
   def infer_related_cloud_tenant_ems_refs_api!
     tenants.each do |tenant|
-      add_simple_target(:cloud_tenants, tenant.try(:parent_id)) unless tenant.try(:parent_id).blank?
+      add_simple_target!(:cloud_tenants, tenant.try(:parent_id)) unless tenant.try(:parent_id).blank?
```

Comment:
@mansam I've got error here without exclamation mark, so added it, is it ok?

Reply:
@aufi yep!